pbxproj/pbxextensions/ProjectFiles.py

```diff
@@ -428,8 +430,8 @@ def add_folder(self, path, parent=None, excludes=None, recursive=True, create_gr
             if os.path.isfile(full_path) or os.path.splitext(child)[1] in ProjectFiles._SPECIAL_FOLDERS or \
                     not create_groups:
                 # check if the file exists already, if not add it
-                children = self.add_file(full_path, parent, target_name=target_name, force=False, tree=TreeType.GROUP,
-                                         file_options=file_options)
+                children = self.add_file(full_path, name=None, parent=parent, target_name=target_name,
```

Comment:
`name=None` is redundant
```suggestion
                children = self.add_file(full_path, parent=parent, target_name=target_name,
```